packages/r2frida/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nowsecure/r2frida
+TERMUX_PKG_DESCRIPTION="Frida IO plugin for radare2"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="5.8.6"
+TERMUX_PKG_SRCURL=https://github.com/nowsecure/r2frida/archive/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=c65979581dc514e58639b5c9492ed21780370446fad6d8cfcaaffe25e86366f1
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS="radare2"
+TERMUX_PKG_BUILD_DEPENDS="radare2"
+
+termux_step_pre_configure () {
+	if [ $TERMUX_ARCH != "aarch64" ]; then
+		termux_error_exit "Unsupported arch: $TERMUX_ARCH"
+	fi
+	unset CXXFLAGS
+	unset CPPFLAGS
+	unset LDFLAGS
+	unset CFLAGS
+}
+
+termux_step_configure () {
+	./configure --prefix="${TERMUX_PREFIX}" --with-precompiled-agent
```

Comment:
Add `--with-precompiled-agent` to `TERMUX_PKG_EXTRA_CONFIGURE_ARGS` instead and avoid overriding termux_step_configure()

Reply:
ideally i would like to remove this flag. but this file is generated by the r2frida-compile program, which is compiled before the plugin.

Reply:
I did it to avoid the warnings about the use of unhandled flags, but this is harmless:

```
WARNING: Unknown flag '--disable-dependency-tracking'.
WARNING: Unknown flag '--disable-rpath'.
WARNING: Unknown flag '--disable-rpath-hack'.
WARNING: Unknown flag '--disable-nls'.
WARNING: Unknown flag '--enable-shared'.
WARNING: Unknown flag '--enable-static'.
```